spring-boot-project/spring-boot-docs/src/docs/asciidoc/using-spring-boot.adoc

```diff
@@ -404,12 +404,12 @@ If you find that specific auto-configuration classes that you do not want are be
 	import org.springframework.context.annotation.*;
 
 	@Configuration(proxyBeanMethods = false)
-	@EnableAutoConfiguration(exclude={DataSourceAutoConfiguration.class})
+	@SpringBootApplication(exclude={DataSourceAutoConfiguration.class})
 	public class MyConfiguration {
 	}
 ----
 
-If the class is not on the classpath, you can use the `excludeName` attribute of the annotation and specify the fully qualified name instead.
+If you choose to use `@EnableAutoConfiguration` rather than `@SpringBootApplication`, please note that it also has an `exlcude` attribute that can be used. If the class is not on the classpath, you can use the `excludeName` attribute of the annotation and specify the fully qualified name instead.
```

Comment:
`exlcude` is misspelled - should be `exclude`

Reply:
Thank you @scottfrederick .
Updated in the latest commit